src/components/product.js

```diff
@@ -406,14 +388,14 @@ export default class Product extends Component {
    * @return {Object}
    */
   get trackingInfo() {
-    if (!this.model.selectedVariant) {
+    if (!this.selectedVariant) {
```

Comment:
may be a little more straight forward to avoid the negated condition. Just `if (this.selectedVariant) { return { stuff } } else { return  {} }`